hvac/v1/__init__.py

```diff
@@ -271,6 +271,15 @@ def ha_status(self):
         """
         return self._get('/v1/sys/leader').json()
 
+    def read_lease(self, lease_id):
+        """
+        PUT /sys/leases/lookup
```

Comment:
Could you please expand these docstrings a bit to capture the relevant information from the linked Vault documentation? E.g.: https://github.com/ianunruh/hvac/blob/master/hvac/v1/__init__.py#L670-L680

Reply:
Added the additional docstrings..

about the integration tests: Looks like the `lease_id` is only available with dynamic secrets, from something like the `database` secrets engine. 

Enabling the `database` secrets engine and using it in the integration tests means that we need to spin up a mysql/postgres database to retreive a dynamic secret.